src/docs/Contributing/NewPattern/SubmitNewPattern/index.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Submit a new pattern'
+order: 10
+---
+
+The creation of a new pattern on MOZAIC is usually done via the following workflow:
+
+## 1. Create a new branch
+
+Creating your new pattern starts with creating a github branch dedicated to it.  
+This branch is ideally named as follows: `pattern-xxx`.
+
+**xxx** refers to the name of your pattern _(exemple : **pattern-flags** ou **pattern-ratingstars**)_.
+
+
+## 2. Create your folder structure using Plop
+
+Plop is a simple generator.  
+We use it to save bootstrapping time when creating new pattern.
+
+To use it, simply type in your terminal `npx plop pattern`
+
+Then the CLI will ask you 3 questions : 
+
+- the pattern name : use a Capital first letter and no spaces (ex: Buttons)
+- the pattern path : from the `src/docs` directory with no `/` at the end (ex: Components)
```

Comment:
maybe something need to be changed into the plop js file to set path to src docs ?